sig-contributor-experience/annual-report-2021.md

```diff
@@ -4,79 +4,105 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - [Migrating K-Dev to a managed kubernetes.io account](https://github.com/kubernetes/community/issues/5877)
+   - Steering Election with Elekto
+      - Major change to how we do elections
+      - There were some hiccups but overall it's a step in the right direction
```

Comment:
I don't think you need this line unless you go into what hiccups there were and why its important to mention that here